packages/react-kit/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { FooterProps } from './props';
+
+/**
+ * @see {@link https://live-storybook-react-kit.appa.pantheon.site/?path=/docs/footer-footer--docs}
+ */
+export const Footer = ({ Logo, Link }: FooterProps) => {
```

Comment:
I would add a prop for `className` so styles could be passed in to the `<footer>`